sycl/include/CL/sycl/detail/defines_elementary.hpp

```diff
@@ -87,10 +87,13 @@
 #endif
 #endif // __SYCL_FALLTHROUGH
 
+// Stringify an argument to pass it in _Pragma directive below.
+#ifndef __STRINGIFY
+#define __STRINGIFY(x) #x
+
```

Comment:
Why not `#endif // __STRINGIFY` right here? I don't see why the rest needs to be gated (e.g., if a user defines `__STRINGIFY` before including this file, suddenly they don't get `__SYCL_UNROLL` and that seems surprising.)

Reply:
Applied, thanks!